tests/conftest.py

```diff
@@ -29,6 +29,21 @@ def basic_wim() -> Iterator[BinaryIO]:
     yield from open_file_gz("data/basic.wim.gz")
 
 
+@pytest.fixture
+def test4k_wim() -> Iterator[BinaryIO]:
```

Comment:
```suggestion
def basic_wim_4k() -> Iterator[BinaryIO]:
```
Or rename everything to `wim_xxx`?